tests/pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
```

Comment:
Created this to resolve PytestUnknownMarkWarning being generated: https://stackoverflow.com/questions/60806473/pytestunknownmarkwarning-unknown-pytest-mark-xxx-is-this-a-typo

Reply:
sounds good